doc/source/api/metadata.rst

```diff
@@ -10,20 +10,117 @@ Goals and Scope
 
 The metadata API provides information on the primary data objects
 available via the GA4GH API, and facilities to organize primary data
-objects.
+objects. Main goals of the metadata definitions are to provide
 
-The current metadata API is immature and will evolve in future.
+* standardized use of common attributes/values
+* a schema of how (core) objects relate to each other
 
+The  metadata API is under continuous development.
 
-Datasets
-@@@@@@@@
 
+Common Attribute Names and Formats
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
 
-All GA4GH data objects are part of a *dataset*. A dataset is a
+Throughout the schema definitions, a consistent use of common attributes should
+be enforced. The following list should serve as guidance for schema developers.
+
+
+========================= ======================================================
+Attribute                 Note
+========================= ======================================================
+id                        the objects ID, used for references; (at least) locally unique
+name                      a more descriptive object label or legacy ID; should *not* be used as reference
+accessions                an ``ARRAY`` containing other names, IDs, URIs of the object
+description               a string describing aspects of the object; *not* a list or nested object itself
+recordCreateTime          time at which this record was created in ISO 8601 (see **Date and Time...**)
+recordUpdateTime          time at which this record was updated in ISO 8601 (see **Date and Time...**)
+========================= ======================================================
+
+Date and Time Format Specifications
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+Date and time formats are specified as ISO8601 compatible strings, both for
+time points as well as for intervals and durations.
+The required granularity has to be specified as part of the respective
+attributes' documentations.
+
+===========
+Time points
+===========
+
+The specification of a time point is given through the concatenation of
+
+* a date in YYYY-MM-DD
+* the designator "T" indicating a following time description
+* the time of day in HH:MM:SS.SSS form, where "SSS" represents a decimal fraction of a second
+* a time zone offset in relation to UTC
+
+**Examples**
+
+* year (YYYY)
+    2015
+
+* date (e.g. date of birth) in YYYY-MM-DD
+    2015-02-10
+
+* time stamp in milliseconds in YYYY-MM-DDTHH:MM:SS.SSS
+    2015-02-10T00:03:42.123Z
+
+**Implementations**
+
+* recordCreateTime (G2P, metadata)
+* recordUpdateTime (G2P, metadata)
+* many proposed in the metadata branch
```

Comment:
> Drop the section Implementations; this will be quickly out of date and confusing

But then, I have always been told that things don’t get into master if they aren’t called anywhere?

Removed.